src/style/document-header.less

```diff
@@ -59,13 +59,54 @@
 .document-header {
     height: @toolbar-button-size;
     line-height: @toolbar-button-size;
-    max-width: 100%;
-    width: 100%;
+    flex-grow: 10;
     position: relative;
     overflow: hidden;
     display: flex;
 }
 
+.generator-header {
+    height: @toolbar-button-size;
+    line-height: @toolbar-button-size;
+    max-width: 5rem;
+    width: 5rem;
+    display: flex;
+    flex-grow: 1;
+}
+
+.generator-header-buttons { 
+    height: 100%;
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: flex-end;
+}
+
+.generator-header-buttons .button-simple.button-plus {
+        margin-right: 2rem;
+
+        & svg:hover {
+            fill: @item;
+        }
+
+        &.button-simple__disabled {
```

Comment:
related to my other comment a few lines down ... isn't this redundant to that separate disabled-svg block?